app/components/frame-viewer.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import LoadingIndicator from '../components/loading-indicator';
+import getSubjectLocation from '../lib/get-subject-location';
+import VideoPlayer from './video-player';
+import PanZoom from './pan-zoom';
+import TextViewer from './text-viewer';
+
+const SUBJECT_STYLE = { display: 'block' };
```

Comment:
Since this style isn't changing because of a conditional, can you convert it to a class instead?